packages/api/src/utils/logs.js

```diff
@@ -0,0 +1,286 @@
+import { nanoid } from 'nanoid/non-secure'
+
+const logtailApiURL = 'https://in.logtail.com/'
```

Comment:
Should we move this to the constants file?

Reply:
this is very specific to logtail and does not change, so i think it can stay here